instrumentation/httpclient/httpclient.gradle

```diff
@@ -0,0 +1,12 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_11
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+// FIXME: how to pass muzzle (only for jdk 11+)
+//muzzle {
```

Comment:
@malafeev Go ahead and restore this after merging master :)

Reply:
done